nzedb/Contents.php

```diff
@@ -108,6 +125,9 @@ public function getForMenuByTypeAndRole($id, $role)
 		return $arr;
 	}
 
+	/**
+	 * @return bool|\nzedb\Content
```

Comment:
Put the main type first. Also it's not bool, it should be false as true will not be returned